packages/epics/src/agreements/plugins/components/common/token-payout-field-array.tsx

```diff
@@ -25,11 +25,13 @@ export interface Token {
 interface TokenPayoutFieldArrayProps {
   tokens: Token[];
   name?: string;
+  label?: string;
 }
 
 export const TokenPayoutFieldArray = ({
   tokens,
   name = 'payouts',
+  label,
```

Comment:
Rather put default value here
```suggestion
  label = 'Payment Request',
```